subiquity/server/server.py

```diff
@@ -552,8 +553,93 @@ async def apply_autoinstall_config(self, context):
             await controller.apply_autoinstall_config()
             await controller.configured()
 
+    def _strip_controller_keys(
+        self,
+        autoinstall_config: dict[str, Any],
+    ) -> dict[str, Any]:
+        """Removes all unkown keys from autoinstall config."""
```

Comment:
Comment is inverted, since we're returning autoinstall with known controllers removed, we're actually returning the unknowns